### DISCUSSION TITLE
10.11.2023

### DISCUSSION THREAD

**Discussion by ticogarcia** [charlieforward9/ATHLEAT#37] [Team Meetings]
We decided on data we'll be requesting from each API (Strava and Myfitnesspal)

Myfitnesspal
- Date
- Meal (Breafast, lunch, dinner)
- Calories
- Macros 
    - (fat, protein, carbs)

Strava

- Kilojoules (we'll convert to calories)
- Elapsed time
- Sport type
- Distance

**Comment by charlieforward9** [comment]
MFP API Links
* https://myfitnesspalapi.com/docs/diary-get/
* https://myfitnesspalapi.com/docs/subscription-post/

Strava API Links
* https://developers.strava.com/docs/reference/#api-Activities-getLoggedInAthleteActivities
